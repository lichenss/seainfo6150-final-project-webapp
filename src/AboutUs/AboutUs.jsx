import React from 'react'
import Footer from '../Footer/Footer'
import Header from '../Header/Header'
import styles from './AboutUs.module.css'
const AboutUs = () => {
    return (
        <>
            <Header />
            <section className={styles.section}>
            <div className={styles.container}>  
            <div className={styles.titleContainer}>
            <h2 className={styles.title}>About Us</h2>
            </div>
            <div className={styles.descriptionContainer}>
            <p className={styles.content}>We envision a world where anyone, anywhere can transform their life by accessing the worldâ€™s best learning experience.</p>
            <p className={styles.content}>Developer:Chen Liu</p>
            </div>
            </div>
                </section>
            <Footer />
        </>
    )
}
export default AboutUs

        