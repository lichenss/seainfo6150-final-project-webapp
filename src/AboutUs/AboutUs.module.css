.section {
    padding-top: 70px;
    display: flex;
    flex-direction: column;
    place-items: center;
    padding-bottom: 3rem;
    background-color: rgb(229, 231, 233);
}
.content{
    font-family: 'Roboto', sans-serif;
}
@media screen and (min-width:600px){

}
@media screen and (min-width:900px){
    
}
@media screen and (min-width:1200px){
    
}