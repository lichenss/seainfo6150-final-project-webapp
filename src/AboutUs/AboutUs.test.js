import React from "react";
import AboutUs from "./AboutUs";
import { render } from "@testing-library/react";
import { BrowserRouter } from "react-router-dom";
describe("AboutUs tests", () => {
    it("renders correctly", () => {
      const { container } = render(
      <BrowserRouter>
      <AboutUs />
      </BrowserRouter>
      );
      expect(container).toMatchSnapshot();
    });
  });