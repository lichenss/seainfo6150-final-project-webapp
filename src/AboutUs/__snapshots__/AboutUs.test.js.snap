// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AboutUs tests renders correctly 1`] = `
<div>
  <div>
    <header
      class="header"
    >
      <div
        class="navbar"
      >
        <button
          class="logo"
        />
        <div>
          <a
            class="catLink"
            href="/category/computer-science"
          >
            Computer Science
          </a>
          <a
            class="catLink"
            href="/category/business"
          >
            Business
          </a>
          <a
            class="catLink"
            href="/category/data-science"
          >
            Data Science
          </a>
          <a
            class="catLink"
            href="/category/arts-and-humanities"
          >
            Arts and Humanities
          </a>
          <a
            class="catLink"
            href="/category/physical-science-and-engineering"
          >
            Physical Science and Engineering
          </a>
          <a
            class="catLink"
            href="/category/math-and-logic"
          >
            Math and Logic
          </a>
        </div>
        <div
          class="links"
        >
          <a
            class="barLink"
            href="/log-in"
          >
            Log In
          </a>
          <a
            class="barLink"
            href="/sign-up"
          >
            Sign Up
          </a>
        </div>
      </div>
    </header>
  </div>
  <section
    class="section"
  >
    <div
      class="container"
    >
      <div
        class="titleContainer"
      >
        <h2
          class="title"
        >
          About Us
        </h2>
      </div>
      <div
        class="descriptionContainer"
      >
        <p
          class="content"
        >
          We envision a world where anyone, anywhere can transform their life by accessing the worldâ€™s best learning experience.
        </p>
        <p
          class="content"
        >
          Developer:Chen Liu
        </p>
      </div>
    </div>
  </section>
  <footer
    class="footer"
  >
    <div
      class="links"
    >
      <a
        class="link"
        href="/contact-us"
      >
        <h3>
          Contact Us
        </h3>
      </a>
      <a
        class="link"
        href="/about-us"
      >
        <h3>
          About Us
        </h3>
      </a>
    </div>
  </footer>
</div>
`;
